mixer/test/client/mixer_internal_fail/mixer_internal_fail_test.go

```diff
@@ -24,7 +24,6 @@ import (
 
 // Mixer server returns INTERNAL failure.
 func TestMixerInternalFail(t *testing.T) {
-	t.Skip("issue https://github.com/istio/istio/issues/4264")
```

Comment:
The tests run in parallel and start from epoch 0. How does this fix the flakiness?

Reply:
But they will use different base-id.  Usually this is how mixer clien tests work

1) start envoy with conf1,  do some tests
2) Kill envoy, and restart ( envoy with conf2, and some some more tests. 

Before this change, they use the same base-id and same epoch=0, some UDS fails randomly

Now I use different base-id for each restart of Envoy.  It will use different UDS paths.

Reply:
This test is executed many times in parallel for race/coverage. We need to exclude it from that test run, or make the test behave nicely in that setting.

Reply:
base-id are for parallel runs (each run has different base-id), in each run the restart epoch start from 0, so even it is executed many times in parallel, it should work.